projects/ecc-diff-fuzzer/Dockerfile

```diff
@@ -16,10 +16,12 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder
 MAINTAINER p.antoine@catenacyber.fr
-RUN apt-get update && apt-get install -y make cmake
+RUN apt-get update && apt-get install -y make cmake curl bzip2 autoconf automake libtool sed
 RUN git clone --depth 1 https://github.com/catenacyber/elliptic-curve-differential-fuzzer.git ecfuzzer
 RUN git clone --depth 1 https://github.com/ARMmbed/mbedtls.git mbedtls
 RUN git clone --depth 1 https://github.com/ANSSI-FR/libecc.git libecc
 RUN git clone --depth 1 https://github.com/openssl/openssl.git openssl
+RUN curl -O https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.32.tar.bz2
```

Comment:
I used to think that we usually do either `RUN wget` or `ADD`, e.g. https://github.com/google/oss-fuzz/blob/01b5b6bed08acfdfeb6d5e3bf66f9bc80ad4d509/projects/ffmpeg/Dockerfile#L31

But probably this is fine too, since you used it in other Dockerfiles already

Reply:
ok